### PR TITLE
UCR based dropdowns in case-search screen

### PR DESCRIPTION
**Pull request by sravfeyn**
## Summary
https://dimagi-dev.atlassian.net/browse/USH-631

## Feature Flag
USH Case Search Addons

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan
https://dimagi-dev.atlassian.net/browse/QA-2883

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/fixtures/fixturegenerators.py [hunk 1]
```diff
@@ -49,6 +49,32 @@ def item_lists_by_domain(domain):
...
+        for report_config in module.report_configs
+    ]
+    ret = list()
+    for config in report_configs:
+        uri = 'jr://fixture/commcare-reports:%s' % (config.uuid)
+        ret.append({
+            'id': config.uuid,
+            'uri': uri,
+            'path': "/rows/row",
+            'name': config.header.get('en'),
```
Comment (orangejenny) [inline corehq/apps/fixtures/fixturegenerators.py:71]: Since you have the app, can you use the default language here instead of English?

**Review by orangejenny — COMMENTED** [review]
Just minor comments.

File: corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html [hunk 2]
```diff
@@ -66,7 +66,8 @@ <h4 class="modal-title">
                         <option value="">{% trans "Text" %}</option>
                         <option value="daterange">{% trans "Date Range" %}</option>
                         <option value="barcode_scan">{% trans "Barcode" %}</option>
-                        <option value="fixture">{% trans "Lookup Table Selection" %}</option>
+                        <option value="lookup_table_fixture">{% trans "Lookup Table Selection" %}</option>
+                        <option value="report_fixture">{% trans "Mobile UCR Selection" %}</option>
```
[the remark this answers is not in the document]
Comment (sravfeyn) [reply corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html:70]: Sure, good idea.

File: corehq/apps/app_manager/static/app_manager/js/details/case_claim.js [hunk 3]
```diff
@@ -29,7 +32,12 @@ hqDefine("app_manager/js/details/case_claim", function () {
 
         self.lookupTableNodeset = ko.pureComputed({
             write: function (value) {
-                self.nodeset(value);
+                if (value === undefined) {
+                    self.nodeset(null);
```
[the remark this answers is not in the document]
Comment (sravfeyn) [reply corehq/apps/app_manager/static/app_manager/js/details/case_claim.js:36]: The existing UI was differentiating between null and undefined. So I had to go with this.

File: corehq/apps/fixtures/fixturegenerators.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/fixtures/fixturegenerators.py]: Sure, added.

File: corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (sravfeyn) [reply corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html]: Sure.

**Review by orangejenny — APPROVED** [review]

**Comment by sravfeyn** [comment]
Hey @orangejenny this is now QA complete, I have added couple more commits now. Would you please check if this is good to go?

**Comment by orangejenny** [comment]
Sweet, I'll look over the recent commits today.

**Review by orangejenny — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]

**Pull request merged**